lib/resolver.js

```diff
@@ -91,6 +95,10 @@ const resolver = async function ({ config, criteria, protocols, defaults, overri
 
     const importsResolved = await resolveProtocols(resolvedCriteria, {
         import(key) {
+            if (!basedir) {
+                console.log('@vrbo/determination: No basedir set, defaulting to "process.cwd()" for resolving relative json imports.');
+                basedir = process.cwd();
+            }
```

Comment:
if line  76 - 79 and line 98 -101 all are base dir related,  baseDir is used only  line 102.  i would  put those two together.

Reply:
Sadly, line 77 isn't basedir-related. If I moved line 78 down, I'd have to repeat the type check conditional.